packages/@dcl/dcl-rollup/ecs.config.ts

```diff
@@ -71,9 +35,38 @@ const config: RollupOptions = {
       extend: true,
       sourcemap: 'hidden',
       compact: true,
-      plugins: [terser({})]
+      plugins: [terser({ format: { comments: false } })]
     }
+  ],
+  plugins: [
+    resolve({
+      preferBuiltins: false,
+      browser: true
+    }),
+    commonjs({
+      exclude: 'node_modules',
+      ignoreGlobal: true
+    }),
+    {
+      name: 'api-extractor',
+      writeBundle() {
+        return apiExtractorConfig(packageJsonPath, !PROD)
+      }
+    }
+  ]
+}
+
+const ecsConfig: RollupOptions = {
+  ...basicRollupConfig,
+  plugins: [
+    typescript({
+      tsconfig: './tsconfig.json',
+      compilerOptions: {
+        declarationDir: 'types'
```

Comment:
why doing this? Declarations should always be in the same directory as the compiled .js to be imported by non-sdk ts-projects